guides/common/modules/proc_registering-a-host.adoc

```diff
@@ -34,6 +34,12 @@ ifndef::satellite,orcharhino[]
 Specifying an operating system is required when you register machines without `subscription-manager`, such as Debian or Ubuntu.
 endif::[]
 . Optional: From the *{SmartProxy}* list, select the {SmartProxy} to register hosts through.
+[NOTE]
```

Comment:
You need a "+" (list continuation) between the step and the note so that the note is indented and the following steps are numbered correctly.